src/app/core/services/error-handling.service.ts

```diff
@@ -29,6 +30,12 @@ export class ErrorHandlingService implements ErrorHandler {
     }
   }
 
+  private addAutoClose<T>(snackBarRef: MatSnackBarRef<T>) {
+    setTimeout(() => {
+      snackBarRef.dismiss();
```

Comment:
Actually we can be more defensive here by making:
`snackBarRef?.dimiss();`
However, this is not yet supported in the current `CATcher` node version (12)